registry/pluginloader/pluginloader.go

```diff
@@ -0,0 +1,14 @@
+// +build go1.8
+
+package pluginloader
+
+import (
+	"path/filepath"
+	"plugin"
+)
+
+// LoadPlugin loads a pluigin with the given name and platform specifix suffix
```

Comment:
Maybe add a note here that it is expected that plugins will register themselves upon load and are available after. I think this is really cool that it just works like that :)